ui/src/app/pages/lesson-module/lesson-module.component.css

```diff
@@ -0,0 +1,21 @@
+.question-and-answer-container {
+     width: 80vw;
+     height: 80vh;
+     margin: 0 auto;
+     text-align: center;
+}
+
+button {
```

Comment:
Be careful with these sort of global targets. The CSS in one component on the page can affect other components if you don't include a container prepend.